plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeParquetStatisticsUtils.java

```diff
@@ -110,7 +110,13 @@ public static Object jsonValueToTrinoValue(Type type, @Nullable Object jsonValue
             return (long) (int) jsonValue;
         }
         if (type == BIGINT) {
-            return (long) (int) jsonValue;
+            if (jsonValue instanceof Long) {
+                return (long) jsonValue;
```

Comment:
nit (IntelliJ eye candy) : can we add here `//noinspection RedundantCast`

same as here 

https://github.com/trinodb/trino/blob/01270cafb4a4bb0b82f859380171f47e460f51f5/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTypes.java#L146-L149